src/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>To-Do List</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap/5.3.0/css/bootstrap.min.css"
+    />
+
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="to-do-list-con" id="todo-list">to do</div>
```

Comment:
As the previous code reviewer requested please kindly follow the wireframe design of the to list given to you by microverse.Kindly follow the [captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) and make sure your lists looks almost identical to [it](https://www.youtube.com/watch?v=AcUd-_Yjjqg) as well.

This is your design: 

![Screenshot 2023-06-16 at 7 05 31 AM](https://github.com/najibullahjafari/To-do-list/assets/79506105/4a3891ff-7d70-45e1-8e60-0d429bdf43da)

Please kindly stick to this design:
![246279361-3dcfede1-4bb5-4b67-a918-29affc6ceb79](https://github.com/najibullahjafari/To-do-list/assets/79506105/845e5f3c-c6c5-42b0-bd23-b65180adc683)

Its good to be creative but again it is important to follow design rules kindly always pay attention to detail. 😄